torchci/lib/drciUtils.ts

```diff
@@ -22,22 +22,34 @@ export const BOT_COMMANDS_WIKI_URL =
 export const FLAKY_RULES_JSON =
   "https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/flaky-rules.json";
 
-export function formDrciHeader(prNum: number): string {
-    return `## :link: Helpful Links
+export function formDrciHeader(owner: string, repo: string, prNum: number): string {
+    // For PyTorch only
+    if (repo === REPO) {
```

Comment:
Please check that org is correct
```suggestion
    if (isPyTorchPyTorch(owner, repo)) {
```